Lib/test/test_ctypes/test_dlerror.py

```diff
@@ -119,5 +118,66 @@ def test_null_dlsym(self):
             self.assertEqual(os.read(pipe_r, 2), b'OK')
 
 
+@unittest.skipUnless(os.name != 'nt', 'test requires dlerror() calls')
+class TestLocalization(unittest.TestCase):
+
+    @staticmethod
+    def configure_locales(func):
+        return test.support.run_with_locale(
```

Comment:
Do you want `run_with_locales`, the decorator?
`run_with_locale` is a context manager.

Reply:
Oh, never mind, it's both. Sorry for the noise.

Reply:
I've actually changed the test so that all locales are tested and not only the first locale that could be found.

Reply:
Hm, I'm not sure what the intended difference is. But, `run_with_locales` uses `subTest`, so at least it gives better error reporting.

Reply:
- `run_with_locale` only uses the first locale that is correct
- `run_with_locales` runs for all specified locales

Reply:
It is both a decorator and a context manager.

The difference is that `run_with_locales` runs the test with all locales and can only be used as a decorator, `run_with_locale` only runs the test with the first available locale and can also be used as a context manager.

In this case it is enough to use `run_with_locale`.

Reply:
Oh.. so you still want to use `run_with_locale`. Mmh, ok.

Reply:
Most tests use `run_with_locale`. I added `run_with_locales` recently specially for tests which should be run for several locales.

Perhaps in these cases we should use a different approach and continue to run the test for different locales until we find one with translations, and mark the test skipped if no such locale is found. I'll think about this idea if I have a chance. For now, you can use what you prefer.

Reply:
(I'll keep `run_with_locale` since I don't want yet another edit and since it's probably fine to only test a single locale for those specific tests)